media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch

```diff
@@ -0,0 +1,26 @@
+From 8964c64644349c7f232234b51423b2ec5e7c894e Mon Sep 17 00:00:00 2001
```

Comment:
No upstream link?

Reply:
Sorry, I didn't submit the patch upstream back then. Created upstream PR, linked in the patch and ran scrub-patch on the file.

Reply:
Do I need all the PR link, currently I've added the patch-diff (thinking of preservation, although github PR's are preserved I think)

Reply:
I normally include the "regular" PR link instead so it's easier to see the discussion but this is fine too, thanks